stories/grid.stories.tsx

```diff
@@ -29,6 +29,27 @@ export function GridWellPlate() {
       rows={8}
       columns={12}
       displayAsGrid
+      headerStyle={({ position, label }) => {
+        if (label === '') {
```

Comment:
I would rather check for row and column being -1, it's much easier to read that it's for the upper left header cell